assets/css/main.scss

```diff
@@ -1,3 +1,38 @@
 body {
-  color: red;
+}
+
+.my-team {
+  & {
+    width: 80%;
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
```

Comment:
This could simply be:
```
.my-team {
  width: 80%;
  margin: 0 auto;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
}
```

Reply:
As I wanted  to have same indentation for all sub-elements